buildSrc/src/main/kotlin/com/virtuslab/gitmachete/buildsrc/UpdateEapBuildNumber.kt

```diff
@@ -0,0 +1,69 @@
+package com.virtuslab.gitmachete.buildsrc
+
+import org.gradle.api.DefaultTask
+import org.gradle.api.tasks.Input
+import org.gradle.api.tasks.Internal
+import org.gradle.api.tasks.Optional
+import org.gradle.api.tasks.TaskAction
+import org.gradle.api.tasks.options.Option
+import org.gradle.util.VersionNumber
+import org.jsoup.Jsoup
+import org.jsoup.nodes.Document
+import org.jsoup.nodes.Element
+import org.jsoup.select.Elements
+
+import java.util.regex.Matcher
+import java.util.regex.Pattern
+
+open class UpdateEapBuildNumber : DefaultTask() {
```

Comment:
Does it need to be `open`?

Reply:
Apparently yes, this is the compilation output without 'open':
<img width="567" alt="image" src="https://user-images.githubusercontent.com/61919032/181496915-7d4c9d7a-8d69-4125-adb9-cb41a5dc26b6.png">